app.js

```diff
@@ -0,0 +1,45 @@
+var express = require('express');
```

Comment:
It looks like express generator is still using var, but in ES6 it's considered good practice to use `let` and `const` instead.

[Link](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)